Celeste.Mod.mm/Mod/Everest/Everest.Updater.cs

```diff
@@ -368,13 +372,24 @@ public class Source {
                     if (Type.GetType("Mono.Runtime") != null) {
                         installer.StartInfo.FileName = "mono";
                         installer.StartInfo.Arguments = $"\"{installerPath}\"";
-                        if (File.Exists("/bin/sh")) {
-                            installer.StartInfo.FileName = "/bin/sh";
-                            installer.StartInfo.Arguments = $"-c \"cd '{extractedPath}'; mono MiniInstaller.exe\"";
+                        if (File.Exists("/bin/bash")) {
+                            installer.StartInfo.FileName = "/bin/bash";
+                            installer.StartInfo.Arguments = $"-c \"cd '{extractedPath}'; unset MONO_PATH LD_LIBRARY_PATH LC_ALL MONO_CONFIG; /usr/bin/mono MiniInstaller.exe &> log.txt\"";
```

Comment:
Fixed in 84f78d78fd42d2ad90f535b6cd76f5632f18aaef, environment variables are unset to clean up the mono environment so that it finds the system libraries rather than conflicting with Celeste's. (starting mono in a mono process without a clean environment is not a fun time)